index.html

```diff
@@ -231,9 +237,10 @@ <h2 class="CANPOY">CANPOY</h2>
           alt=""
         />
       </div>
+    <div class="desk-continer">
       <div class="top">
         <h2 class="first-text">About <br /><span>Myself</span></h2>
-        <p class="second-text">
+        <p class="second-text second-text1">
```

Comment:
- [ ] As I noticed, you still have the same issues. When you use a `position: absolute`, ensure that the parent element has a `position: relative`. This helps the child to be aligned inside its parent element. If you still struggling with implementing it, you can visit this [link](https://www.w3schools.com/css/css_positioning.asp) to get more understanding of the CSS Layout position.
- Hint: It maybe looks nice on your desktop screen, However, you can use your chrome inspect screen mode to check different screen sizes.

- Here is your desktop design:
    
![image](https://user-images.githubusercontent.com/80612925/171969263-0ab99437-183f-48c6-93a8-5a8fa9a9dbad.png)

- The Figma design
![image](https://user-images.githubusercontent.com/80612925/171969369-cc32c885-7f74-45db-bc63-0f7a6a4fab6b.png)

Reply:
believe me i checked the projec on the google developer tools with all kind of desktop devices and i didnot see something like that!
what are you using for checking the project????? what size ? what device ?
tell me please @Kandy-Peter

Reply:
Hi @Alzubair98 ,
 I used Chrome as well, and that is what I god. I also used [https://raw.githack.com](https://raw.githack.com/) and I'm getting the same result.

Reply:
i will send you a video on slack 
showing you all the models
@Kandy-Peter

Reply:
Hey @Alzubair98,
You can do it. I would like to see how you proceed.

Reply:
https://www.loom.com/share/a9a12f4e983b4503848e33e99b648931

Reply:
kindly check this

Reply:
I got it, lt me check it here too

Reply:
Hello @Alzubair98!

I've just realized that the problem was resolved.
I am sorry to mislead you. I haven't done that on purpose :blush:

Reply:
You'll get an answer as soon as possible to resolve this issues

Reply:
dear kandy 
I'm sure you didn't done this on purpose , you know mistakes happen all the time 
its okay 
thank you for your understanding

Reply:
@Kandy-Peter 💯

Reply:
Thank you for your understanding 🤝